### PR TITLE
Fill in details on Docker use

### PR DESCRIPTION
**Pull request by ramen**
Make the Docker docs more comprehensive, and incorporate content from our internal Wiki.

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -1,26 +1,265 @@
...
+- Make sure Docker is installed and running on your machine.
+- Make sure you're using the correct command to run the Docker image.
+- Check the Docker logs for the container to see if there are any error
+  messages.
+- If you're using the GPU version of the Docker image, make sure you have a
+  compatible NVIDIA GPU and that the NVIDIA container toolkit is installed on
+  your machine.
+
+If you're still having trouble, feel free to reach out to the ReaSpeech team
+for help.
```
Comment (mikeylove) [inline docs/docker.md:146]: do we have a discord perma-invite we can toss in here?

**Review by mikeylove — APPROVED** [review]
great expansion of detail for understanding more about the docker side...but also kind of overwhelming? this is _a lot_ of information. i can see further iterations of breaking it down in some ways and expanding in others, but this is a solid bit of progress.

**Comment by ramen** [comment]
> great expansion of detail for understanding more about the docker side...but also kind of overwhelming? this is _a lot_ of information. i can see further iterations of breaking it down in some ways and expanding in others, but this is a solid bit of progress.

Valid point. I tried to put the essential stuff at the top, and get into the gritty details toward the bottom. I could add a section heading like "Advanced Usage", or move some content to development.md and/or a new docker_advanced.md

File: docs/docker.md [hunk 2]
```diff
@@ -1,26 +1,265 @@
...
+
+To stop the Docker container, you'll need to know the container ID. You can get
+a list of running containers and their IDs by running:
 
 ```sh
-# Build and Run for CPU
-docker build -t reaspeech .
-docker run -d -p 9000:9000 --name reaspeech reaspeech
+docker ps
+```
```
[the remark this answers is not in the document]
Comment (ramen) [reply docs/docker.md:91]: Yeah, great idea. I think that should be at the top, before getting into any command line examples.

File: docs/docker.md (hunk 1 above)
Comment (ramen) [reply docs/docker.md:149]: Probably, but we should mention it on the main README first. Going to wait on this until we're ready.

**Pull request merged**